packages/functional-tests/lib/accountManager.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import crypto from 'crypto';
+
+export const BLOCKED_EMAIL_PREFIX = 'blocked{id}';
+export const BOUNCED_EMAIL_PREFIX = 'bounced{id}';
+export const FORCE_PWD_EMAIL_PREFIX = 'forcepwdchange{id}';
+export const SIGNIN_EMAIL_PREFIX = 'signin{id}';
+export const SIGNUP_REACT_EMAIL_PREFIX = 'signup_react{id}';
+export const SYNC_EMAIL_PREFIX = 'sync{id}';
+type EmailPrefix =
+  | typeof BLOCKED_EMAIL_PREFIX
+  | typeof BOUNCED_EMAIL_PREFIX
+  | typeof FORCE_PWD_EMAIL_PREFIX
+  | typeof SIGNIN_EMAIL_PREFIX
+  | typeof SIGNUP_REACT_EMAIL_PREFIX
+  | typeof SYNC_EMAIL_PREFIX;
+
+type Account = {
+  email: string;
+  password: string;
+};
+
+export const PASSWORD = 'passwordzxcv';
+
+export class AccountManager {
+  accounts: Account[] = [];
+
+  create(prefix: EmailPrefix = SIGNIN_EMAIL_PREFIX): Account {
```

Comment:
@pdehaan I'll take your suggestion to break this down as follows once we get the tests passing.
```

  createBlocked(): Account {
    return this.create(BLOCKED_EMAIL_PREFIX);
  }

  createBounced(): Account {
    return this.create(BOUNCED_EMAIL_PREFIX);
  }
```